.travis.yml

```diff
@@ -5,11 +5,12 @@ install:
 jobs:
   include:
     - stage: unit tests
-      script: yarn test:ci
+      script: yarn run test:ci
     - stage: build
-      script: yarn build
-      after_success: npm run coverage
+      script: yarn run build
+      after_success: yarn run coverage
```

Comment:
i would recommend combining the coverage upload with the tests

Reply:
Yeah, I think that would be good.

Reply:
moved in 006ee969237217d23d95942249deace5d23a0ff7